tinyxml2.cpp

```diff
@@ -75,10 +75,12 @@ distribution.
 				const int required = _vsnprintf(str, len, format, va);
 				delete[] str;
 				if ( required != -1 ) {
+					TIXMLASSERT( required >= 0 );

```

Comment:
@ariccio `required` can also be `-1` which means output doesn't fit. That's why there's an `if ( required != -1 )` in the first place.